tutorecommerce/plugin.py

```diff
@@ -180,3 +183,35 @@
 tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(
     list(config.get("overrides", {}).items())
 )
+
+
+@tutor_hooks.Filters.APP_PUBLIC_HOSTS.add()
+def _print_ecommerce_public_hosts(hosts: list[str], context_name: t.Literal["local", "dev"]) -> list[str]:
+    if context_name == "dev":
+        hosts += ["ecommerce.{{ LMS_HOST }}:8130"]
```

Comment:
Replace "ecommerce.{{ LMS_HOST }}" by "{{ ECOMMERCE_HOST }}" (on this line and the next one)